DESCRIPTION

```diff
@@ -19,11 +19,14 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    cluster,
+    VennDiagram,
+    igraph
 Suggests:
     testthat,
     BiocStyle,
     knitr,
     rmarkdown
-RoxygenNote: 6.1.1
+RoxygenNote: 6.1.1.9000
```

Comment:
Why is patch level .9000 required? What happens if you use `6.1.1`. If we need to require the higher version, is this potentially a breaking change for the other packages?

Reply:
I don't know. I deleted the .9000 and nothing bad seems to have happened yet

Reply:
I called roxygen2::roxygenize() and the .9000 has re-appeared. I have no Idea what it means.

Reply:
Ok - I can't tell either ... if it's not something you need, please delete and we'll keep an eye on it.

Reply:
Actually - I deleted it here.